docs/pages/database-access/guides/gui-clients.mdx

```diff
@@ -353,3 +353,75 @@ Click `Add Redis Database`.
 Congratulations! You have just connected to your Redis instance.
 
 ![Redis Insight Connected](../../../img/database-access/guides/redis/redisinsight-connected.png)
+
+
+## Snowflake - JetBrains (IntelliJ, Goland, DataGrip, PyCharm etc.)
+
+<Admonition type="note">
+  Snowflake integration works only in the authenticated proxy mode. Remember to add `--tunnel` flag when starting the proxy, example:
+  ```
+  tsh proxy db --tunnel --port 2000 snowflake
+  ```
+</Admonition>
+
+In "Database Explorer" click and the add button and then pick "Data Source" and pick "Snowflake":
```

Comment:
```suggestion
In "Database Explorer" click the "add" button, pick "Data Source", and then pick "Snowflake":
```
I think it would be a bit clearer to use quotation marks around "add," even though the button doesn't actually say "add."